.claude/commands/debug.md

```
@@ -48,6 +72,81 @@ Protocol Phases:
+
+Required Capabilities: backend, infra, performance
+Estimated Token Cost: ~6k-15k tokens
+
+Recommendation: Use --team for automated spawning, or --agents 3 for manual control.
+
+Commands:
+  /debug start "..." --team           # Accept recommendation
+  /debug start "..." --agents 2       # Override agent count
+  /debug start "..."                  # Single agent (simpler)
+```
+
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)